mobility-ransack.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.files        = Dir['{lib/**/*,[A-Z]*}']
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'ransack', '>= 1.8.0', '< 2.1'
+  spec.add_dependency 'ransack', '>= 1.8.0'
```

Comment:
Instead of making this a totally open-ended dependency, how about: `'>= 1.8.0', '<3'` ?

Reply:
Done!